config.json

```
@@ -1564,6 +1564,16 @@
       "topics": [
         "arrays"
       ]
+    },
+    {
+      "slug": "knapsack",
+      "uuid": "b0301d0b-d97a-4043-bd82-ba1edf8c1b16",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 1,
+      "topics": [
+        "algorithms"
```

There are a couple of other topics that are applicable to any reasonable solution.

```suggestion
        "algorithms",
        "lists",
        "loops"
```